src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -97,9 +99,14 @@ TriggeredSubscription::~TriggeredSubscription()
 * TriggeredSubscription class is shared for NGSI9 and NGSI10 subscriptions, so it is better
 * to keep expressions (an artifact for NGSI10) out of the constructor, in its independent fill
 * method
-*
 */
-void TriggeredSubscription::fillExpression (const std::string& q, const std::string& georel, const std::string& geometry, const std::string& coords)
+void TriggeredSubscription::fillExpression
+(
+  const std::string&  q,
+  const std::string&  georel,
+  const std::string&  geometry,
+  const std::string&  coords
+)
 {
```

Comment:
Yes, with noCache we have no choice but to parse the qStringFilter every time.
The change I did in TriggeredSubscription was simply to add a pointer to the parsed qStringFilter, pointing to the subCache.  When noCache is used, this pointer is NULL